Hide Queries fields

artem-zinnatullin opened the pull request:
Closes #321 

@nikitin-da PTAL

nikitin-da commented:
Emission hasn't caught, may be we need to increase waiting time?

nikitin-da commented on storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/DeleteQuery.java:
```diff
@@ -11,40 +11,63 @@
-     * Passing null will return all rows for the given table
+     * Formatted as an SQL {@code WHERE} clause (excluding the {@code WHERE} itself).
+     * <p/>
+     * If it's {@code null} — Query will retrieve all rows for the given table.
```
Do you mean `Query will delete all rows`?

artem-zinnatullin replied:
Yep, my bad, I'll fix this

ср, 13 мая 2015, 13:50, Dmitrii Nikitin <notifications@github.com>:

> In
> storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/DeleteQuery.java
> <https://github.com/pushtorefresh/storio/pull/325#discussion_r30219929>:
>
> >       * <p/>
> > -     * Passing null will return all rows for the given table
> > +     * Formatted as an SQL {@code WHERE} clause (excluding the {@code WHERE} itself).
> > +     * <p/>
> > +     * If it's {@code null} — Query will retrieve all rows for the given table.
>
> Do you mean Query will delete all rows?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/325/files#r30219929>.
>

artem-zinnatullin replied:
Fixed!

artem-zinnatullin commented:
Damn, I'll create a bug for this, I face this on my Core i7 Desktop from
time to time, I don't think that it's the problem of performance, I'll
check the code of TestObserver, maybe I made a bug in notifications
implementation.

ср, 13 мая 2015, 13:48, Dmitrii Nikitin <notifications@github.com>:

> Emission hasn't caught, may be we need to increase waiting time?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/325#issuecomment-101623125>.
>

artem-zinnatullin commented:
Rebased and squashed into one commit. @nikitin-da if notifyAboutChangesConcurrently will fail again and it'll be the only failed test — merge this without worries, I'll fix it later.

nikitin-da commented:
Tests not found

nikitin-da commented:
retest this please

artem-zinnatullin commented:
retest this please
